### PR TITLE
update dependencies & fix source in contraband-scala directory is not published

### PR DESCRIPTION
**Pull request by ohze**

### CODE REVIEW

**Review by eed3si9n — APPROVED** [review]
Thanks!

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,8 @@ project/pf.sbt
 pf.sbt
 
 rsync.sh
+/.idea/
+/target/
+/*/target/
+/project/project/target/
+/shaded/asynchttpclient/target/
```
Comment (dwijnand) [inline .gitignore:9]: You can generalise for all target directories at any level with `target/`.

File: build.sbt [hunk 2]
```diff
@@ -74,6 +74,7 @@ lazy val core = (project in file("core")).
     name := "gigahorse-core",
     libraryDependencies ++= Seq(sslConfig, reactiveStreams, slf4jApi, scalatest % Test),
     managedSourceDirectories in Compile += (sourceDirectory in Compile).value / "contraband-scala",
+    unmanagedSourceDirectories in Compile += (sourceDirectory in Compile).value / "contraband-scala",
```
Comment (dwijnand) [inline build.sbt:77]: The duplication with `managedSourceDirectories` is a little suspicious to me.. Is it really ok to have it in both?

**Review by dwijnand — COMMENTED** [review]
Wow. Nice effort! Left a couple of comments.

File: build.sbt (hunk 2 above)
Comment (eed3si9n) [reply build.sbt:77]: It's for the inclusion of managed source, so maybe a better solution might be to figure out where the source jars are made and fix the mapping.

**Pull request merged**